chapter_03.md

```diff
@@ -61,6 +72,16 @@ If you prefer to write your tests with TypeScript or Babel you can pick one of t
 
 Since we've already written our automation script, we can press `n` here as our objective will be to transfer that script into a test.
 
+If you select `y` you will get the following questions
+
+> What should be the location of your spec files? ("suggested/path/test/specs/\*_/_.ts")
+
+The auto-populated test files will be generated at this path. If you want you can provide your own.
+
+> Do you want to use page objects (https://martinfowler.com/bliki/PageObject.html)?
+
+Select if you want to use page objects.
+
```

Comment:
Should we include this, since we tell the user to continue with `n`?

Reply:
I'll remove it, tnx